### PR TITLE
[ELY-2601] declared  newMap on a separate line in PasswordKeyStoreSpi

### PR DESCRIPTION
**Pull request by nasonawa**
[ELY-2601](https://issues.redhat.com/browse/ELY-2601) newMap on a separate line in PasswordKeyStoreSpi

### CODE REVIEW

**Comment by PrarthonaPaul** [comment]
Hi @nasonawa 
Thank you for the PR! 
Would you be able to add the ELY issue number to the commit message? You can use `git commit --ammend` to edit your commit message. 
Additionally, please add the link to the ELY issue in the description.

**Comment by nasonawa** [comment]
Hi @PrarthonaPaul 
I have made the suggested Changes, 
let me know if it's correct or not

**Review by PrarthonaPaul — APPROVED** [review]
Looks good to me! 
Thanks @nasonawa

File: credential/base/src/main/java/org/wildfly/security/keystore/PasswordKeyStoreSpi.java [hunk 1]
```diff
@@ -91,7 +91,9 @@ public void engineSetEntry(final String alias, final KeyStore.Entry entry, final
         if (protParam != null) {
             throw log.keyCannotBeProtected(alias);
         }
-        HashMap<String, PasswordEntry> map, newMap;
+        HashMap<String, PasswordEntry> map;
+        HashMap<String,PasswordEntry> newMap;
```
Comment (skyllarr) [inline credential/base/src/main/java/org/wildfly/security/keystore/PasswordKeyStoreSpi.java]: @nasonawa Justa t total minor, can you please fix the formatting by adding a space before the `PasswordEntry` of newMap initialization? Thank you!

**Comment by nasonawa** [comment]
Hi @Skyllarr 
made suggested the changes. 
let me know if it's correct or not

**Review by skyllarr — APPROVED** [review]

**Comment by skyllarr** [comment]
@nasonawa Looks great, thank you!

**Pull request merged**